File: 3rdPartyServices/StudentServices/CollaborativeQuiz/SocialLearningGame/SocialLearningGame/Entities/UserAnsweredQ.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SocialLearningGame.Entities
{
    public class UserAnsweredQ
    {
        public int id { get; set; }
        public String userJid { get; set; }
        public int questionID { get; set; }
        public Boolean answeredCorrect { get; set; }
    }
}

File: 3rdPartyServices/StudentServices/CollaborativeQuiz/SocialLearningServer/src/main/java/ac/hw/services/collabquiz/entities/UserAnsweredQ.java
package ac.hw.services.collabquiz.entities;

import java.util.HashMap;

import javax.persistence.*;

@Entity
@Table(name="UserAnsweredQ")
public class UserAnsweredQ {


	private int id;
	private String userJid;
	private int questionID;
	private boolean answeredCorrect;
	
	public UserAnsweredQ() {
		
	}
	
	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	
	public String getUserJid() {
		return userJid;
	}
	public void setUserJid(String userJid) {
		this.userJid = userJid;
	}
	public int getQuestionID() {
		return questionID;
	}

	public void setQuestionID(int questionID) {
		this.questionID = questionID;
	}

	public boolean isAnsweredCorrect() {
		return answeredCorrect;
	}

	public void setAnsweredCorrect(boolean answeredCorrect) {
		this.answeredCorrect = answeredCorrect;
	}

}
